src/components/CarouselContainer.ts

```diff
@@ -120,21 +122,25 @@ export default class CarouselContainer extends Component<CarouselContainerProps,
             });
             this.setState({ images, isLoading: false });
         } else if (this.props.dataSource === "XPath" && this.props.imagesEntity) {
```

Comment:
Re-write as L124 to L127
` else if (this.props.dataSource === "XPath" && this.props.imagesEntity && mxObject) {`
`   this.fetchImagesByXPath(mxObject);`
`}`

Reply:
Done